src/main/java/seedu/moneygowhere/storage/LocalStorage.java

```diff
@@ -145,14 +205,89 @@ private Expense createExpense(Node node) throws LocalStorageLoadDataInputError {
         return new Expense(name, dateTime, description, amount, category, remark, currency);
     }
 
+    /**
+     * This method takes in a target node and convert it into a Target object.
+     *
+     * @param node containing information from a target
+     * @return a target created with data by node
+     */
```

Comment:
I believe that your Javadoc comments may not be following the form defined in the coding standard. Do take a look at the coding standard at https://se-education.org/guides/conventions/java/basic.html

Reply:
Noted. Will work on it.